server/src/test/scala/services/ServerSDateSpec.scala

```diff
@@ -154,13 +155,13 @@ class ServerSDateSpec extends Specification {
     }
   }
 
-  "When asking for getLocalLastMidnight" >> {
+  "When asking for getLocalLastMidnight with a non-UTC timezone" >> {
 
-    val date = SDate("2020-04-02T23:00:00Z")
-    val expected = SDate("2020-04-02T23:00:00Z")
+    val date = SDate("2020-04-02", europeLondonTimeZone)
+    val expected = LocalDate(2020, 4, 2)
 
-    s"Given ${date.toISOString()} then I should get ${expected.toISOString()}" >> {
-      val result = date.getLocalLastMidnight
+    s"Given ${date.toISOString()} then I should get $expected" >> {
+      val result = date.addMinutes(0).getLocalLastMidnight.toLocalDate
```

Comment:
do we need to add `0` minutes here?

Reply:
no i'll remove that. i was trying to recreate a bug, and thought adding zero minutes might have triggered it, but it didn't so this change is certainly not needed